### PR TITLE
updated kory's bio

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

**Comment by trel** [comment]
looks good to me - we'll update the title 'real soon now'.

**Comment by korydraughn** [comment]
I'm thinking about adding a line about new duties.

**Comment by trel** [comment]
roadmap... and i can remove it from mine...

File: content/pages/team.html [hunk 1]
```diff
@@ -34,14 +34,13 @@ <h3>Terrell Russell<br/>Executive Director</h3>
...
                     <br/>
-                    Since joining the team as a Research Software Developer back in February 2018, Kory has helped
-                    design and implement various pieces of software across the iRODS ecosystem. Major work includes,
-                    but isn't limited to, Rule Engine Plugin Continuation, Parallel Transfer, Logical Quotas Rule
-                    Engine Plugin, C++17 Filesystem library, and NFSRODS. Kory is interested in distributed systems,
-                    performance, design, algorithms, and open source software. In his new role, he aims to help improve
-                    the quality of the iRODS software, ecosystem for developers, and community. Kory holds a B.S. in
-                    Computer Science from East Carolina University and a M.S. in Computer Science from New Jersey
-                    Institute of Technology.
+                    Since joining the team as a Research Software Developer in February 2018, Kory has helped
```
Comment (trel) [inline content/pages/team.html:37]: ecosystem twice...    

nothing wrong with it - just noticed.

**Comment by korydraughn** [comment]
This is ready to be merged.

**Pull request merged**